docker/create_chroot_from_image.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+image=$1
+created=$(docker inspect --format='{{.Created}}' $image)
+timestamp="${created:0:4}${created:5:2}${created:8:2}-${created:11:2}${created:14:2}"
+gadgetron_hash=$(docker run --rm $image cut -c-8 /opt/code/gadgetron_sha1.txt)
+gtprep_hash=$(docker run --rm $image [ -f /opt/code/gtprep_sha1.txt ] && $(cut -c-8 /opt/code/gtprep_sha1.txt))
```

Comment:
I did the test. "[ -f /opt/code/gtprep_sha1.txt ] && $(cut -c-8 /opt/code/gtprep_sha1.txt)" does not work. Just "cut -c-8 /opt/code/gtprep_sha1.txt" will work fine.